src/readii/image_processing.py

```
@@ -181,15 +184,23 @@ def displayImageSlice(
     # If image is a simple ITK image, convert to array for display
     if type(image) == sitk.Image:
         image = sitk.GetArrayFromImage(image)
+        # Once converted to an array, sliceDim is always first
+        sliceDim = "first"
 
     # Get min and max value from image to define range in display
     if dispMin == None:
```

Ruff (0.8.0)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)